test/io/jesi/backpack/test/strict_test.cljc

```diff
@@ -0,0 +1,273 @@
+
+(deftest use-fixtures-test
+
+  (testing "use-fixtures"
+    (is (= #?(:clj  '(clojure.test/use-fixtures :each identity)
+              :cljs '(def cljs-test-each-fixtures [identity]))
+           (macroexpand '(io.jesi.backpack.test.strict/use-fixtures :each identity))))))
+
+(deftest is-test
+
+  (testing "is"
+
+    #?(:clj (testing "is a macro"
+              (is (macro? `strict/is))))
+
+    (testing "expands to a `clojure.test/is` statement"
+      (is-macro= #?(:clj  '(try
+                             (clojure.core/let [value# true]
+                               (if value#
+                                 (clojure.test/do-report {:type     :pass
+                                                          :expected 'true
+                                                          :actual   value#
+                                                          :message  nil})
+                                 (clojure.test/do-report {:type     :fail
+                                                          :expected 'true
+                                                          :actual   value#
+                                                          :message  nil}))
+                               value#)
+                             (catch java.lang.Throwable t#
+                               (clojure.test/do-report {:type     :error
+                                                        :expected 'true
+                                                        :actual   t#
+                                                        :message  nil})))
+                    ;TODO use env/transform once it supports converting catch clauses
+                    :cljs '(try
+                             (clojure.core/let [value# true]
+                               (if value#
+                                 (cljs.test/do-report {:type     :pass
+                                                       :expected 'true
+                                                       :actual   value#
+                                                       :message  nil})
+                                 (cljs.test/do-report {:type     :fail
+                                                       :expected 'true
+                                                       :actual   value#
+                                                       :message  nil}))
+                               value#)
+                             (catch :default t#
+                               (cljs.test/do-report {:type     :error
+                                                     :expected 'true
+                                                     :actual   t#
+                                                     :message  nil}))))
+                 (macroexpand '(io.jesi.backpack.test.strict/is true))))
+
+    (testing "takes"
+
+      (testing "an optional message string"
+        (strict/is true "true")
+        (strict/is true (str true))))
```

Comment:
This was broken before...
A macro `:pre` receives quotes symbols, never a string